main.cpp

```diff
@@ -32,7 +32,10 @@ int main()
           Solid::HyperelasticSolver<2> solid(triangulation, params);
           solid.run();
           */
-          Utils::GridCreator::flow_around_cylinder(triangulation);
+          
+          //Utils::GridCreator::flow_around_cylinder(triangulation);
+          Utils::GridCreator::pipe_flow(triangulation);
```

Comment:
I think adding `pipe_flow` function is unnecessary because it does nothing more than a call to `GridGenerator::subdivided_hyper_rectangle`.

Reply:
Changed